docs/optimization-framework/ab-testing.md

```diff
@@ -1,19 +1,14 @@
-# Experiments
+# A/B Tests
 
-Backed by [native Altis Analytics](./native.md), Experiments are a powerful tool for optimizing content and measuring the effectiveness of changes to the site.
+A/B Tests provides the tools to carry out randomized experiments using different variations of content features, eg: titles and featured images, and/or different variations of content blocks.
```

Comment:
Not sure on the use of the word randomized just here, to me it read like the experiment was random, rather than the assignment of users to different variants for example.
```suggestion
A/B Tests provides the tools to carry out experiments using different variations of content features, eg: titles and featured images, and/or different variations of content blocks.
```